example/components/Resistor/examples/SeriesExample.m

```diff
@@ -39,10 +39,10 @@ - (void)viewDidLoad {
 
 @end
 
-@implementation SeriesResistorExample (CatalogByConvention)
+@implementation SeriesExample (CatalogByConvention)
 
 + (NSArray<NSString *> *)catalogBreadcrumbs {
```

Comment:
Will need to update the method signature here to reflect the flexibility of this API.